cmdb.js

```diff
@@ -22,12 +23,12 @@ function cmdb(config) {
  * @param {Object} [body] - An object to send to the API
  * @returns {Promise<Object>} The data received from CMDB (JSON-decoded)
  */
-cmdb.prototype._fetch = function _fetch(locals, path, method, body) {
+cmdb.prototype._fetch = function _fetch(locals, path, method, body, timeout = 6000) {
 	var params = {
 		headers: {
 			apikey: this.apikey,
 		},
-		timeout: 6000,
+		timeout: timeout,
```

Comment:
My inclusion of a timeout parameter here may have been a bit misleading, due to this: https://github.com/matthew-andrews/isomorphic-fetch/issues/48